src/backend/joanie/core/management/commands/delete_bulk_users.py

```diff
@@ -0,0 +1,58 @@
+"""
+Management command getting all users to delete from a redis set and dispatching a task to delete
+them.
+"""
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from django_redis import get_redis_connection
+
+from joanie.core.tasks import delete_user
+
+
+class Command(BaseCommand):
+    """
+    This command is responsible to delete all the users references in redis in the set
+    with key settings.JOANIE_DELETE_BULK_USERS_REDIS_SET_KEY.
+
+    Without the --limit option, all users in the set are deleted at once. Be careful, if hundreds
+    of thousands users are in the set, it can be a long process.
```

Comment:
```suggestion
    WARNING: 
    Without the --limit option, all users in the set are deleted at once. Be careful, if hundreds
    of thousands users are in the set, it can be a long process.
```